src/devices/Seesaw/SeesawAdc.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.Seesaw
+{
+    public partial class Seesaw : IDisposable
+    {
+        /// <summary>
+        /// Reads the value of an analog pin.
+        /// </summary>
+        /// <param name="pin">The pin number in the devices numbering scheme.</param>
+        /// <returns>A value between 0..1023 that represents the analog value.</returns> 
+        public UInt16 AnalogRead(byte pin)
+        {
+            const int AdcConversionDelay = 500; //500 uSeconds delay between read and write to allow Adc conversion time.
```

Comment:
Add `microseconds` somewhere in the name, then not sure if comment will be any more useful

Reply:
Done in the latest PR update.